src/docs/product/alerts-notifications/issue-alerts.mdx

```diff
@@ -0,0 +1,85 @@
+| New event detection | Sentry detects a new event with level {**match**} {**level**}. A **match**: equal to, less than or equal to, greater than or equal to. A **level**: fatal, error, warning, info, debug, or sample. |
+
+## Actions
+
+Actions are the configurations that determine what will happen when an alert is triggered. The following actions are available:
+
+- Send an email to either [Issue Owners](/product/error-monitoring/issue-owners/), Team, or [Member](/accounts/membership/#member).
+- Send a notification to an [integration](/product/integrations/), which includes:
+
+  - [PagerDuty](/product/integrations/integrations/pagerduty/)
+  - [Slack](/product/integrations/integrations/slack/)
+  - [Microsoft Teams](/product/integrations/integrations/msteams/)
+  - Legacy integrations
+  - Integrations built on the [Integration Platform](/product/integrations/integration-platform/)
+
+  If no legacy integrations or integrations built using the integration platform are enabled, this option is hidden.
+
+### Issue Owners
+
+By default, [issue owners](/product/error-monitoring/issue-owners/) receive alert emails. If an issue owner is not configured or not found, the email will either not send or send to all project members as defined in **Project Settings > Issue Owners**.
+
+![A toggle indicating if all users are issue owners or not.](issue_owners2.png)
+
+## Rate Limit
+
+The rate limit determines how frequently an issue alerts. After an issue triggers an alert, Sentry won't check the conditions and won't execute the actions for that issue until the rate limit period passes. The limit is set to perform the action according to one of these intervals:
+
+- minutes: 5, 10, 30, 60
+- hours: 3, 12, 24
+- one week or 30 days
+
+For example, if an issue violates alert conditions multiple times in a 30-min period, but your frequency threshold is 30 minutes, you’ll only get one alert.
+
+![A dropdown that adjusts the frequency of an alert.](alert_frequency.png)
+
+## Environment
+
+Environment control allows you to specify which environment qualifies for your issue alert:
+
+- All Environments
+- Any of your defined (and not hidden) environments.
+
+If you select "All Environments", Sentry checks individually for each environment, rather than combined events across environments.
+
+![A dropdown allowing you to choose an environment to attach to an alert.](alert_environment.png)
+
+## Ignore
```

Comment:
@adhiraj Support had mentioned that customers get confused about "Ignore" so I thought it best to give it its own section. Let me know if there's a better place for this that will alleviate the customer questions.

Reply:
Maybe change title. Maybe "Issues Set to Ignore"